docs/references/calfits_memo.tex

```diff
@@ -1,175 +1,179 @@
-\documentclass[11pt, oneside, english]{article}   	% use "amsart" instead of "article" for AMSLaTeX format

```

Comment:
Apparently there were trailing spaces on all these lines that my editor removed. To make it easier to review the actual changes, use the "Hide whitespace" option under the gear icon in the "Files Changed" tab.